src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportGenerator;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String DELIMITER = ",";
+
+    @Override
+    public List<String> createReport(Map<String, Integer> fruitTransactionDtoList) {

```

Comment:
Bad name + Better to create report using the direct Storage access instead of passing it as a parameter here
```suggestion
    public List<String> createReport() {
```